### DISCUSSION TITLE
Comment ajouter ces emojis a Gboard?

### DISCUSSION THREAD

**Discussion by Zardangus** [community/community#31704] [Other Feature Feedback, Questions, & Ideas]
Bonjour,

Je suis tombé sur ce lien :
https://github.com/alcor/emoji-supply

C'est un projet très sympa, mais je n'arrive pas à ajouter les emojis dans Gboard.

Quelqu'un peut m'expliquer comment faire s'il vous plaît ?

Merci bien.

**Comment by Ebuube** [comment]
Je ne sais pas aussi
